octue/cloud/pub_sub/topic.py

```diff
@@ -41,12 +43,17 @@ def create(self, allow_existing=False):
         :return None:
         """
         if not allow_existing:
-            self.service.publisher.create_topic(name=self.path)
+            self.service.publisher.create_topic(
+                request=Topic_(name=self.path, labels={"created": str(datetime.now().timestamp()).split(".")[0]})
```

Comment:
I'd create the timestamp as a separate variable so it's clear what it is (especially since that fairly gnarly code is used twice)...
```py
posix_timestamp_no_decimals = str(datetime.now().timestamp()).split(".")[0]
```